includes/Admin/Dashboard.php

```diff
@@ -303,6 +303,7 @@ public function load_stories_dashboard(): void {
 			'/web-stories/v1/settings/',
 			'/web-stories/v1/publisher-logos/',
 			'/web-stories/v1/users/me/',
+			'/web-stories/v1/taxonomies/',
```

Comment:
```suggestion
			'/web-stories/v1/taxonomies/?' . build_query(
				[
					'type'    => $this->story_post_type->get_slug(),
					'context' => 'edit',
				]
			),
```

The query args here need to match the query args used in `getTaxonomies()` in `packages/wp-dashboard/src/api/taxonomies.js`

This way, we can save 1 HTTP request when loading the dashboard.

To verify this chang, open the dev tools and see that there is no HTTP request to `/web-stories/v1/taxonomies/` when opening the dashboard

Reply:
You will also need to change the preloading as well.

Reply:
This _is_ the preloading section.

This suggestion fixes the preloading.